`make statusgo-ios` fails with Xcode-15

# Problem

`make statusgo-ios` fails on macos with Xcode 15 like this : 

```
github.com/status-im/status-go/mobile
gobind
error: cannot compute path of binary 'Path(str: "/private/tmp/gomobile-work/ios/iphoneos/Statusgo.framework/
Versions/A/Statusgo")' relative to that of '/tmp/gomobile-work/ios/iphoneos/Statusgo.framework'
/nix/store/s2lzniglbn16321k88s1qhnf6pi12qxk-gomobile-unstable-2022-05-18/bin/gomobile: 
xcodebuild -create-xcframework -framework /tmp/gomobile-work/ios/iphoneos/Statusgo.framework 
-framework /tmp/gomobile-work/iossimulator/iphonesimulator/Statusgo.framework -framework 
/tmp/gomobile-work/iossimulator/iphonesimulator/Statusgo.framework -output 
build/bin/Statusgo.xcframework failed: exit status 70

make: *** [statusgo-ios] Error 1
```

--- a/nix/overlay.nix
+++ b/nix/overlay.nix
@@ -62,6 +62,10 @@ in rec {
         url = "https://github.com/golang/mobile/commit/406ed3a7b8e44dc32844953647b49696d8847d51.patch";
         sha256 = "sha256-dqbYukHkQEw8npOkKykOAzMC3ot/Y4DEuh7fE+ptlr8=";
       })
+      (final.fetchurl { # https://github.com/golang/go/issues/63141
+        url = "https://github.com/golang/mobile/commit/e2f452493d570cfe278e63eccec99e62d4c775e5.patch";
+        sha256 = "sha256-gFcy/Ikh7MzmDx5Tpxe3qCnP36+ZTKU2XkJGH6n5l7Q=";
+      })
     ];
   }));
 }

